### ISSUE TITLE
Add 'how many books are currently in the Open library' as part of the 'Around the Library' stats

### ISSUE THREAD

**Issue by BrittanyBunk** [internetarchive/openlibrary#2398]
It would be nice to see how many books exist in history, and this would be a great starting point. It would get updated, like how the other stats change too.

**Comment by guyjeangilles** [comment]
A cool idea. I'm afraid the legitimacy of the stat would be suspect until the duplicates in OL's data are under control

**Comment by BrittanyBunk** [comment]
@guyjeangilles that's true. A lot of ideas I mention would require a fix of current issues before proceeding. I was thinking of a 28 day count too, but that would still have the same troubles. What should I do with the issue then? Close and reopen or just leave it?

**Comment by guyjeangilles** [comment]
@BrittanyBunk I'd leave it open and let's say it's blocked by current data cleanup efforts

**Comment by BrittanyBunk** [comment]
@guyjeangilles @ScarlettRed https://openlibrary.org/stats it's already here - so that's not an issue. As the duplicates go down, the totals will reflect that. I think it's ok to add it in because of it not being an issue (we could always put a disclaimer in if it's truly an issue).
